mobile/App.tsx

```diff
@@ -1,13 +1,11 @@
 import React from "react";
 import { Provider as PaperProvider } from "react-native-paper";
-import { StatusBar, StyleSheet, Text, View } from "react-native";
+import { StyleSheet, Text, View } from "react-native";
 import Navigation from "./Navigation";
-import { appTheme } from "./constants";
 
 export default function App() {
   return (
-    <PaperProvider theme={appTheme}>
-      <StatusBar backgroundColor={appTheme.colors.primary} />
+    <PaperProvider>
```

Comment:
Oh, we don't need the theme anymore?

Reply:
Added the theme back, check the new commits!